package.json

```diff
@@ -45,5 +48,13 @@
     "nyc": "^15.1.0",
     "ts-node": "^9.1.1",
     "typescript": "^4.0.3"
+  },
+  "dependencies": {
+    "@oclif/command": "^1.8.0",
+    "@oclif/test": "^1.2.8"
+  },
+  "oclif": {
+    "commands": "./src/commands",
```

Comment:
`./src/commands` will contain TypeScript files, I think we'll want this to point to `./build/src/commands`, since that's where tsc will drop our compiled JS files. Can we double check this?

and/or remove the `oclif` property? It seems like we're mainly wiring up the `bin` section versus all of the prepack/postpack work that we could (should?) be doing via npm scripts

Reply:
We need to the oclif property to be able to run the commands., but I agree that we should point it to the compiled js at ./build/src/commands.